eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -3,16 +3,16 @@ parameters:
   BaselineFilePath: ''
   SourceDirectory: $(Build.SourcesDirectory)
   ServiceDirectory: ''
+  ScriptDirectory: 'eng/common/scripts/git-diff-changes.ps1'
```

Comment:
This isn't the right name for this parameter. Did you want it to be only the directory or the full diff script path?

Reply:
Is there any good reason to make this a template parameter?

Reply:
Just give the flexibility. Otherwise, we have to hard-coded the path to step.

Reply:
What flexibility are we gaining here? Would we ever have any cases where we would pass a different script?

Reply:
I don't have any cases for current use.